src/utils.ts

```diff
@@ -110,4 +113,12 @@ export default class Utils {
       }
     }
   }
+
+  static setFailSilentlyFlag(options: Options) {
+    process.env.SECRETS_FOUNDRY_FAIL_SILENTLY = (+(options.failSilently === true)).toString();
```

Comment:
Do we need to do it via an env variable? We again run the risk of overriding someone else's env variable. Can we not just pass the argument and compare ?